logical/src/main/java/org/apache/drill/common/expression/FieldReference.java

```diff
@@ -1,4 +1,4 @@
-/**
```

Comment:
In Java, comments of the form /** ... */ are Javadoc comments; they are copied into the Javadoc generated for the code. If we put the copyright in such comments, it will appear at the top of the Javadoc file for the class.

Most projects do no include a copyright notice at the top of each Javadoc file (at least not in the visible HTML shown to readers.) Our copyright notice is not formatted for Javadoc, so it is unlikely that our goal is to include the notice in Javadoc.

By contrast, the /* ... */ style comment is a simple code comment, not copied into Javadoc. So, to avoid showing the copyright in Javadoc, we use the plain-comment style instead.